### DISCUSSION TITLE
Is it possible to exclude a subdomain from matching URLs?

### DISCUSSION THREAD

**Discussion by stefanobartoletti** [keepassxreboot/keepassxc#10032] [Q&A]
I'm trying to understand if there is a way to exclude a subdomain from matching URLs when using the browser extension.

The use case is like this, I have stored my credentials like this

```
Site 1: 
User: myUsername1
Password: myPassword1
Url: mywebsite.com

Site 2: 
User: myUsername2
Password: myPassword2
Url: subdomain.mywebsite.com
```

This is a common situation when you have different CMS or applications running on the same domain but in different subdomains.

When using autocomplete with the browser extension, you can go in the `Browser integration` tab and add additional domains (so in example, I can add `mywebsite.net` to the list of accepted domains of `Site 1`), but I cannot find how to **exclude** a given subdomain/pattern.

In my previous example, I don't want `Site 1` credentials to show when I visit `subdomain.mywebsite.com`.

I tried to add `!subdomain.mywebsite.com` but it seems that it is not supported.

Is there any way to do this?

**Comment by droidmonkey** [comment]
The only option is to go to the application settings, browser tab, then enable "only return best matching"

**Comment by stefanobartoletti** [comment]
Thanks, it seems that it is enough for my use case.
